project/Dependencies.scala

```diff
@@ -8,16 +8,18 @@
  */
 
 import sbt._
+import Keys._
 
 object Dependencies {
   // Keep in sync with .github CI build
   val Scala212 = "2.12.18"
   val Scala213 = "2.13.12"
-  val ScalaVersions = Seq(Scala212, Scala213)
+  val Scala3 = "3.3.1"
+  val ScalaVersions = Seq(Scala212, Scala213, Scala3)
 
   val PekkoVersion = PekkoCoreDependency.version
 
-  val SlickVersion = "3.3.3"
+  val SlickVersion = "3.5.0-RC1"
```

Comment:
Should we only use this version in scala 3 build？

Reply:
See discussion at https://github.com/slick/slick/discussions/2891#discussioncomment-8550269